### PR TITLE
post: Chapters Video Request

### PR DESCRIPTION
**Pull request by DennyPerez18**
# PyLadies Pull Request
_Please document the following. We will aim to be as timely as possible in processing your pull request, however as this is a volunteer staffed project anticipate at least a week for pull requests to start to be reviewed._ 

## Is this pull requesting adding or updating a [PyLadies chapter website](https://github.com/pyladies/pyladies#to-add-a-new-pyladies-location)? 
 

- [ ] Yes
- [ ] No

## Have you confirmed the website builds locally? See [here](https://github.com/pyladies/pyladies#run-the-site-locally).

- [ ] Yes
- [ ] No

##  Does your pull request pass the `netlify/pyladies/deploy-preview` check?

- [ ] Yes
- [ ] No

# Pull Request Checklist

- [ ] Fill out all sections above
- [ ] Share pull request in PyLadies Slack channel #project-tech-infra

/cc @pyladies/tech-and-infra-admins

### CODE REVIEW

File: www/_posts/2023/2023-11-03-chapters-video-request.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+---
+layout: post.html
+title: "Chapters video request"
```
Comment (Mariatta) [inline www/_posts/2023/2023-11-03-chapters-video-request.md:3]:
```suggestion
title: "🎬 Share Your PyLadies Chapter at PyLadiesCon!"
```

File: www/_posts/2023/2023-11-03-chapters-video-request.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+tags: [PyLadies, Conference, Global]
+author: Denny Perez
+author_link: https://twitter.com/dennyperez18
+---
+
+Chapters video request
```
Comment (Mariatta) [inline www/_posts/2023/2023-11-03-chapters-video-request.md:9]: Let's remove lines 9-12. We will use the subject as the blog post title. So the blog post will just start with "Hello amazing PyLadies!"

File: www/_posts/2023/2023-11-03-chapters-video-request.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+Subject: 🎬 Share Your PyLadies Chapter at PyLadiesCon!
+
+Hello Amazing PyLadies!
```
Comment (Mariatta) [inline www/_posts/2023/2023-11-03-chapters-video-request.md:13]:
```suggestion
Hello amazing PyLadies!
```

**Comment by Mariatta** [comment]
I made some suggestions but otherwise look great

**Comment by DennyPerez18** [comment]
@Mariatta, everything should be fine now 😄.

**Review by Mariatta — APPROVED** [review]
Thanks!

**Pull request merged**